addons/website/views/snippets/s_blockquote.xml

```diff
@@ -2,14 +2,16 @@
 <odoo>
 
 <template name="Blockquote" id="s_blockquote">
-    <blockquote class="s_blockquote s_blockquote_classic w-100 mx-auto o_animable blockquote">
-        <i class="s_blockquote_icon fa fa-1x fa-quote-left bg-o-color-2 rounded"/>
-        <div class="s_blockquote_content bg-100">
-            <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
-            <footer class="blockquote-footer">
-                <img src="/web/image/website.s_blockquote_default_image" class="s_blockquote_avatar img rounded-circle me-2" alt=""/>
-                <span class="s_blockquote_author text-muted"><b>John DOE</b> &#8226; CEO of MyCompany</span>
-            </footer>
+    <blockquote class="s_blockquote s_blockquote_minimalist s_blockquote_small o_cc o_cc2 o_animable position-relative d-flex flex-column gap-4 w-100 mx-auto p-4 fst-normal">
+        <div class="s_blockquote_minimalist_border position-absolute top-50 start-0 translate-middle-y h-100 bg-primary"/>
+        <i class="s_blockquote_icon fa fa-quote-right bg-o-color-1 position-absolute top-0 start-50 translate-middle rounded"/>
+        <lead class="s_blockquote_quote my-auto">" Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services. "</lead>
+        <div class="s_blockquote_infos d-flex gap-2 w-100">
+            <img src="/web/image/website.s_blockquote_default_image" class="s_blockquote_avatar img rounded-circle" alt="Paul Dawson"/>
+            <div class="s_blockquote_author d-flex flex-column justify-content-center small">
```

Comment:
Rather then using `small`, wrap each entry  in a `o_small` class, this way will be editable.
If you're in doubt, re-create the desired result using the editor.
This is what I get...
```xml
<p class="o_default_snippet_text o_small">
    <strong>Bruce BRD<br></strong>
    <font class="text-600">Super boss</font>
</p>
```
...that could become something like this (to be tested)
```xml
<div class="s_blockquote_author d-flex flex-column justify-content-center small">
  <span class="o_small">
    <strong>Bruce BRD</strong><br>
    <span class="text-muted">Super boss</span>
  </p> 
</div>
```

Reply:
> ...that could become something like this (to be tested)
This seems to work well, thanks 👍